packages/spor-button-react/src/Button.tsx

```diff
@@ -8,7 +8,15 @@ import { useTranslation } from "@vygruppen/spor-i18n-react";
 import React from "react";
 import { ButtonSpinner } from "./ButtonSpinner";
 
-type ButtonProps = Exclude<ChakraButtonProps, "colorScheme" | "loadingText">;
+export type ButtonProps = Exclude<
+  ChakraButtonProps,
+  "colorScheme" | "loadingText" | "size" | "variant"
+> & {
+  /** The size of the button. Try not to use the xs size a lot */
+  size?: "xs" | "sm" | "md" | "lg";
+  /** The different variants of a button */
+  variant?: "control" | "primary" | "secondary" | "tertiary" | "additional";
```

Comment:
The `size` and `variant` props were previously generated for us, but it required another build step by consumers. Instead, we just list them out here, as these APIs will be pretty stable for now.